packages/commonwealth/client/scripts/views/modals/WelcomeOnboardModal/steps/PersonalInformationStep/validations.ts

```diff
@@ -0,0 +1,14 @@
+import { VALIDATION_MESSAGES } from 'client/scripts/helpers/formValidationMessages';
+import z from 'zod';
+
+export const personalInformationFormValidation = z.object({
+  username: z
+    .string({ invalid_type_error: VALIDATION_MESSAGES.NO_INPUT })
+    .nonempty({ message: VALIDATION_MESSAGES.NO_INPUT }),
+  email: z.union([
+    z.literal(''),
+    z.string({ invalid_type_error: VALIDATION_MESSAGES.NO_INPUT }).email(),
+  ]),
+  enableAccountNotifications: z.boolean(),
+  enableProductUpdates: z.boolean(),
```

Comment:
shouldn't we mark them as optional?

Reply:
we could. I am actually provided `initialValues={{enableAccountNotifications: false, enableProductUpdates: false}}` to the CWForm for the submitted values to be actually boolean.